library/core/src/cell.rs

```diff
@@ -2270,6 +2273,7 @@ impl<T: DispatchFromDyn<U>, U> DispatchFromDyn<UnsafeCell<U>> for UnsafeCell<T>
 /// See [`UnsafeCell`] for details.
 #[unstable(feature = "sync_unsafe_cell", issue = "95439")]
 #[repr(transparent)]
+#[rustc_diagnostic_item = "sync_unsafe_cell"]
```

Comment:
The CI will most likely fail on this. For some dark magic reasons rustc wants this in the form of

```rust
#[cfg_attr(not(test), rustc_diagnostic_item = "Cell")]
```
[[src](https://github.com/rust-lang/rust/pull/132732/files#diff-75a679ddd1aae61c8b60e45cea1fb213a086caee3600993c68af9f7a09785e9eL307)]

Also I wonder if this should be in CamelCase instead or that doesn't matter too much.

Reply:
The regular unsafe cell symbol is in there as `unsafe_cell`, so I was just following that as an example. I could change one, or both?